Description: remove remote image from README.md to avoid privacy breach
 The README.md is converted to html and is part of the installed docs.
Forwarded: not-needed
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2014-07-29
Index: npm/README.md
===================================================================
--- npm.orig/README.md
+++ npm/README.md
@@ -1,6 +1,6 @@
 npm(1) -- a JavaScript package manager
 ==============================
-[![Build Status](https://img.shields.io/travis/npm/npm/master.svg)](https://travis-ci.org/npm/npm)
+[Build Status](https://travis-ci.org/npm/npm)
 ## SYNOPSIS
 
 This is just enough info to get you up and running.
